README.md

```diff
@@ -123,6 +123,7 @@ GOOGLE_SHEETS_TEMPLATE_FILE_ID=...
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=...
 
 # for Slack integration
+SLACK_BASE_URL=https://my-workspace.slack.com
 SLACK_API_TOKEN=...
```

Comment:
Ideally there should be validation to ensure that the base url matches the SLACK_API_TOKEN for the app.

Reply:
Will add a TODO for that